docs/source/user/file_formats.rst

```diff
@@ -129,6 +127,24 @@ browser and select the “Editor” item in the “Open With” submenu:
     <iframe src="https://www.youtube-nocookie.com/embed/b5oAoVB3Wd4?rel=0&amp;showinfo=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
   </div>
 
+JupyterLab's powerful grid viewer can load files up to the maximum string
```

Comment:
I would delete "powerful" - I think it's sounds too self-aggrandizing in the docs, and would rather let the *user* say "wow, that's powerful"